types/src/transaction/analyzed_transaction.rs

```diff
@@ -131,11 +133,18 @@ impl AnalyzedTransaction {
             // receiver account resource.
             write_hints.push(Self::account_resource_location(receiver_address));
         }
+
+        let read_hints = vec![
```

Comment:
Should we also add the framework code location here?

Reply:
That's a lot of modules, I'm thinking maybe we need to come up with some clever tricks down the road. For example, assuming commonly needed modules for all shards.